src/Collections.ts

```diff
@@ -582,27 +589,26 @@ export abstract class ArrayQueryable<TElement>
         }
     }
 
-    public sequenceEqual(other: IQueryable<TElement>, comparer?: EqualityComparer<TElement>): boolean
+    public sequenceEqual(other: IQueryable<TElement> | Array<TElement>): boolean;
+    public sequenceEqual(other: IQueryable<TElement> | Array<TElement>, comparer: EqualityComparer<TElement>): boolean;
+    public sequenceEqual(other: IQueryable<TElement> | Array<TElement>, comparer?: EqualityComparer<TElement>): boolean
     {
+        if (comparer === undefined || comparer === null)
```

Comment:
`comparer?: EqualityComparer<TElement>` only accepts `undefined`, never `null`. Drop null comparison

(null-accepting version would be: `comparer?: EqualityComparer<TElement> | null` )